docs/troubleshooting.md

```diff
@@ -94,6 +94,18 @@ sudo apt install python3-venv python3-pip
 Reference:
 [Python Packaging User Guide: Installing pip/setuptools/wheel with Linux Package Managers](https://packaging.python.org/guides/installing-using-linux-tools)
 
+## macOS issues
+
+If you want to use a Pipx installed package in a shebang (a common example is the AWS cli),
+you will likely not be able to, because the binary will be under `~/Library/Application Support/pipx`.
```

Comment:
```suggestion
If you want to use a Pipx-installed package in a shebang (a common example is the AWS CLI),
you will likely not be able to, because the binary will be stored under `~/Library/Application Support/pipx/`.
```